frontend/cypress/tests/chartView/chartView_widget.cy.js

```diff
@@ -0,0 +1,38 @@
+describe("Widget Mode Copy and Logo Verification", () => {
+  it("Copies the iframe and checks for logo", () => {
```

Comment:
Could we frame this like a test case (what is the action and what is expected)? Feel free to rephrase

```suggestion
  clipboard button copies iframe tag and iframe source displays logo
```